hat/templates/admin/base.html

```diff
@@ -0,0 +1,12 @@
+{% extends 'admin/base.html' %}
+{% load static %}
+
+
+
+{% block extrastyle %}
+  <link rel="stylesheet" href="{% static 'css/admin-json-widget.css' %}">
```

Comment:
Not really import but FYI  you can set the  CSS via the  JSON_EDITOR_CSS settings (or in the inheritance of the widget, if you want to add a supplementary file to the builtin one)

cf 

https://github.com/jmrivas86/django-json-widget/blob/master/django_json_widget/widgets.py#L15

https://docs.djangoproject.com/en/3.2/topics/forms/media/

Reply:
Well the ting is that I'm not replacing the default JSON_EDITOR_CSS, I'm adding to it .. ?

Reply:
I'll have a look into it because I think it would be better if we dont need the `/hat/templates/admin/base.html` file